src/SocialAuthUserManager.php

```diff
@@ -319,11 +431,13 @@ public function createUser($name, $email) {
     $violations = $new_user->validate();
```

Comment:
Please remove this statement and the if block after it. It is not necessary to validate here.